phoenix-core/src/main/java/org/apache/phoenix/mapreduce/transform/TransformTool.java

```diff
@@ -775,30 +798,58 @@ public void resumeTransform(String[] args, CommandLine cmdLine) throws Exception
         }
 
         runTransform(args, cmdLine);
+
+        // Check if we already have a TransformMonitor task. If we do, remove those and start a new monitor
+        List<Task.TaskRecord> taskRecordList = Task.queryTaskTable(connection, null);
+        for (Task.TaskRecord taskRecord : taskRecordList) {
+            if (taskRecord.getTaskType() == PTable.TaskType.TRANSFORM_MONITOR) {
+                Task.deleteTask(connection.unwrap(PhoenixConnection.class), PTable.TaskType.TRANSFORM_MONITOR, taskRecord.getTimeStamp(), taskRecord.getTenantId(),
```

Comment:
Isn't each TransformMonitor Task only meant for one particular transform record? Shouldn't we only delete the task if taskRecord matches the transform record we're interested in?